tests/data/MerchFactory.ts

```diff
@@ -35,14 +36,13 @@ export class MerchFactory {
     const fake = MerchandiseItemModel.create({
       uuid: uuid(),
       itemName: faker.datatype.hexaDecimal(10),
-      picture: faker.image.cats(),
+      // picture: faker.image.cats(),
       description: faker.lorem.sentences(2),
```

Comment:
nit: if we don't need this code anymore, let's remove it instead of commenting it out

Reply:
tbh I left it there because I thought fake.image.cats() is funny.

Reply:
will remove